ansible/roles/routemgmt/files/installRouteMgmt.sh

```diff
@@ -61,3 +63,27 @@ $WSK_CLI -i --apihost "$APIHOST" action update --auth "$AUTH" "$NAMESPACE/routem
 -a description 'Delete the API' \
 --kind nodejs:default \
 -a web-export true -a final true -a require-whisk-auth true
+
+echo Installing apimgmt package
+$WSK_CLI -i --apihost "$APIHOST" package update --auth "$AUTH"  --shared no "$NAMESPACE/apimgmt" \
+-a description "This package manages the gateway API configuration." \
+-p gwUser "$GW_USER" \
+-p gwPwd "$GW_PWD" \
+-p gwUrl "$GW_HOST" \
```

Comment:
@csantanapr - true, when we completely remove the v1 interface, the `gwUrl` will no longer be needed.  right now, the separate `apimgmt` and `routemgmt` packages leverage the exact same code which requires the `gwUrl` value.  imho, i'd like to handle the v1 removal in a separate pr.